### PR TITLE
Edits to "Component Styles" and "Pipes"

### PR DESCRIPTION
**Pull request by cfranger**
@juleskremer here are my edits for the "Component Styles" page. Just three comments/questions in this one--search for "CF:".

### CODE REVIEW

**Comment by cfranger** [comment]
@juleskremer I added another commit with my edits to the "Pipes" page. Just two comments in this page--search for "CF:"

File: public/docs/ts/latest/guide/component-styles.jade [hunk 1]
```diff
@@ -107,151 +103,151 @@ a(id="special-selectors")
...
   
-  We have several ways to add styles to a component: 
-  * inline in the template HTML
-  * by setting `styles` or `styleUrls` metadata
-  * with CSS imports
+  There are several ways to add styles to a component: 
+  * Inline in the template HTML.
+  * By setting `styles` or `styleUrls` metadata.
+  * With CSS imports.
+  <!-- CF: Should the following H3 sections be organized in the same order as this list? -->
```
Comment (juleskremer) [inline public/docs/ts/latest/guide/component-styles.jade]: @cfranger yes

File: public/docs/ts/latest/guide/component-styles.jade [hunk 2]
```diff
@@ -287,15 +282,17 @@ code-example(format="").
...
 :marked
   These are the styles we wrote, post-processed so that each selector is augmented
+  <!-- CF: I didn't change this "we" because I think it's referring to Angular. --> 
   with `_nghost` or `_ngcontent` attribute selectors. 
-  These extra selectors enable the scoping rules described in this guide.
+  These extra selectors enable the scoping rules described in this page.
   
-  We'll likely live with *emulated* mode until shadow DOM gains traction.
+  We'll likely live with emulated mode until shadow DOM gains traction.
+  <!-- CF: Not sure if this "we" refers to Angular or the reader. Is this sentence necessary at all? -->
```
Comment (juleskremer) [inline public/docs/ts/latest/guide/component-styles.jade]: No, please remove.

File: public/docs/ts/latest/guide/pipes.jade [hunk 3]
```diff
@@ -103,90 +105,88 @@ figure.image-display
...
 
 :marked
-  If we try the <live-example></live-example>,
-  we can probe its behavior by changing the value and the optional exponent in the template.
+  To probe the behavior in the <live-example></live-example>, 
+  change the value and optional exponent in the template.
 
-  ## Power Boost Calculator (extra-credit)
+  ## Power Boost Calculator
+  <!-- CF: I wasn't sure if "Power Boost Calculator" is a proper noun. I left it in title caps. -->
```
Comment (juleskremer) [inline public/docs/ts/latest/guide/pipes.jade]: is ok

File: public/docs/ts/latest/guide/pipes.jade [hunk 4]
```diff
@@ -196,213 +196,213 @@ figure.image-display
...
+  When you can't, you *can* use the impure pipe.
 
 .l-sub-section
   :marked
-    Or we might not use a pipe at all.
+    Or you might not use a pipe at all.
     It may be better to pursue the pipe's purpose with a property of the component,
-    a point we take up later.
+    a point that's discussed later.
+    <!-- CF: Later in this page? In another topic? Should we specify where this is discussed? -->
```
Comment (juleskremer) [inline public/docs/ts/latest/guide/pipes.jade]: refers to line 479

**Comment by cfranger** [comment]
Thanks for your comments, @juleskremer !  My latest commit resolves all your notes.

**Comment by juleskremer** [comment]
Thanks! lgtm

**Pull request merged**